src/js/setup-text-tracks.js

```diff
@@ -25,15 +25,39 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
   const tracksAttached = tracks
     // Map input data to match HTMLTrackElement spec
     // https://developer.mozilla.org/en-US/docs/Web/API/HTMLTrackElement
-    .map((track) => ({
-      dashTrack: track,
-      trackConfig: {
-        label: track.lang,
-        language: track.lang,
-        srclang: track.lang,
-        kind: track.kind
+    .map((track) => {
+      let localizedLabel;
+
+      if (Array.isArray(track.labels)) {
+        for (let i = 0; i < track.labels; i++) {
```

Comment:
Same thing, pretty sure you meant
```suggestion
        for (let i = 0; i < track.labels.length; i++) {
```